### PR TITLE
IBX-411: Users segments

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-411
| Versions      | master, 4.0

### CODE REVIEW

**Review by ciastektk — APPROVED** [review]

File: docs/personalization/perso_configuration.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ To do this, navigate to one of the pages mentioned below and edit the item that
 ## Models
 
 The **Models** page lists all available models and provides detailed information, 
-such as the scenarios that use each model or when the model was last triggered.
+such as the scenarios that use each model or when the model was last triggered or segments.
```
Comment (dabrt) [inline docs/personalization/perso_configuration.md:19]:
```suggestion
such as the scenarios or segments that use each model or when the model was last triggered.
```

File: docs/personalization/recommendation_models.md [hunk 2]
```diff
@@ -211,3 +211,25 @@ The logic used for resolving a submodel is as follows:
     If you do not want to group recommendations based on a certain attribute, 
     remove the attribute parameter from the request. 
     The submodel is then omitted.
+
+## Segments
+
+Segments allow to get personalized content suitable for particular users groups. They compute models based on segment attribute factor.
```
Comment (dabrt) [inline docs/personalization/recommendation_models.md:217]:
```suggestion
With segments, you can get personalized content that is suitable for particular users groups. 
Segments compute models based on segment attribute factor.
```
Is it "segments" or... "models" maybe?

File: docs/personalization/recommendation_models.md [hunk 3]
```diff
@@ -211,3 +211,25 @@ The logic used for resolving a submodel is as follows:
...
+Information with user segment are provided in each event which comes from the tracking script.
```
Comment (dabrt) [inline docs/personalization/recommendation_models.md:218]:
```suggestion
Information about user segment is provided in each event that comes from the tracking script.
```

File: docs/personalization/recommendation_models.md [hunk 4]
```diff
@@ -211,3 +211,25 @@ The logic used for resolving a submodel is as follows:
...
+
+First, make sure you have [enabled personalization](https://doc.ibexa.co/en/latest/guide/personalization/enabling_personalization/#enable-personalization) and [configured item type tracking](https://doc.ibexa.co/en/latest/guide/personalization/recommendation_client/#set-up-item-type-tracking).
```
Comment (dabrt) [inline docs/personalization/recommendation_models.md:220]:
```suggestion
To use segments, you must first [configure item type tracking](https://doc.ibexa.co/en/latest/guide/personalization/recommendation_client/#set-up-item-type-tracking).
```
Enabling perso seems self-explanatory, we're halfway through an article about perso configuration.

File: docs/personalization/recommendation_models.md [hunk 5]
```diff
@@ -211,3 +211,25 @@ The logic used for resolving a submodel is as follows:
...
+
+If your [user role](../site_organization/organizing_the_site.md#permissions) includes 
+the `Segment/All functions`, `Segment group/All functions` Policies, you can configure segment settings in the models according to your requirements.
+To do this, go to the **Models** section and click the **Edit** icon next to a name of the model.
+
+With segments groups you can assign users to different recommendation groups based on data gathered and deliver recommendations to these user groups.
```
Comment (dabrt) [inline docs/personalization/recommendation_models.md:226]:
```suggestion
With segment groups you can assign users to different recommendation groups based on data gathered, and deliver recommendations to these user groups.
```

File: docs/personalization/recommendation_models.md [hunk 6]
```diff
@@ -211,3 +211,25 @@ The logic used for resolving a submodel is as follows:
...
+
+The **Segment** list displays only active segments and is generated from the events collected for relevant history (the actual data from recommendation engine, not what is added using the Back Office).
```
Comment (dabrt) [inline docs/personalization/recommendation_models.md:228]:
```suggestion
The **Segment** list displays active segments only and is generated from the events collected for relevant history (the actual data from the recommendation engine, and not data that you upload through the Back Office).
```

**Review by dabrt — APPROVED** [review]

File: docs/personalization/recommendation_models.md (hunk 6 above)
Comment (juskora) [reply docs/personalization/recommendation_models.md:229]: It's not uploaded, it's manually added.

**Pull request merged**